buildozer/README.md

```diff
@@ -68,6 +68,8 @@ Buildozer supports the following commands(`'command args'`):
     importing the symbols. Before using this, make sure to run
     `buildozer 'fix movePackageToTop'`. Afterwards, consider running
     `buildozer 'fix unusedLoads'`.
+  * `replace_load <path> <[to=]from(s)>`: Similar to `new_load`, but removes
+    existing load statements for the requested symbols before adding new loads.
```

Comment:
Adding an example (in the Examples section below) could be useful.

Reply:
Sounds good. I'll send a PR.